src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -30,11 +30,16 @@ void LoadARFF(const std::string& filename,
   std::ifstream ifs;
   ifs.open(filename, std::ios::in | std::ios::binary);
 
+  // if file is not open throw an error (file not found).
+  if (!ifs.is_open()){
```

Comment:
Tiny style issue, the `{` should be on a new line.

Reply:
Sure